flux_led/fluxled.py

```diff
@@ -662,14 +710,23 @@ def brightness(self):
         for CCT calculate the brightness.
         for ww send led level
         """
-        if self.mode in ["DIM", "ww"]:
-            return int(self.raw_state.warm_white)
-        elif self.mode == "CCT":
+        color_mode = self.color_mode
+        raw_state = self.raw_state
+
+        if color_mode == COLOR_MODE_DIM:
+            return int(raw_state.warm_white)
+        elif color_mode == COLOR_MODE_CCT:
             _, b = self.getWhiteTemperature()
             return b
-        else:
-            _, _, v = colorsys.rgb_to_hsv(*self.getRgb())
-            return v
+
+        _, _, v = colorsys.rgb_to_hsv(*self.getRgb())
+        if color_mode == COLOR_MODE_RGBW:
+            return round((v + raw_state.warm_white) / 2)
+        if color_mode == COLOR_MODE_RGBWW:
+            return round((v + raw_state.warm_white + raw_state.cool_white) / 3)
+
```

Comment:
need to divide the whites by 255.

rgb_to_hsv returns a percentage but, the raw_state is in hex.

Reply:
fixed in c86865e4ff84d2d8e627f4f1846a6d20806f6036

Reply:
per https://stackoverflow.com/questions/42736230/rgb-to-hsv-wrong-in-colorsys